src/components/sidebar/routePathView/routePathView.scss

```diff
@@ -1,26 +1,54 @@
 @import '../../shared/styles/common';
 @import '../../shared/styles/flexbox';
+@import '../../shared/styles/form';
 
 .routePathView {
     padding: 20px;
     overflow-y: auto;
     width: $sidebarWide;
 
-    .routePathTimestamp {
-        font-weight: bold;
+    input {
+        padding: 10px;
+        line-height: 20px;
+        margin-top: 5px;
+        border-radius: 5px;
+        width: 100%;
+    }
+
+    div, input {
+        box-sizing: border-box;
     }
 
-    .subTopic {
+
+    .routePathTimestamp {
         font-weight: bold;
-        color: $mediumGrey;
-        font-size: $smallFontSize;
     }
 
     .routeInformationContainer {
         display: flex;
         flex-direction: row;
         justify-content: space-between;
-        padding-bottom: 10px;
-        border-bottom: 1px solid $lightGrey;
+        padding-top:10px;
+    }
+
+    .checkBoxContainer {
+        margin-top: 15px;
+    }
+
+    .mapCheckboxContainer {
+        margin-top: 10px;
```

Comment:
Centering with margin-top doesn't seem like a good idea. If our checkbox gets larger or font-size changes etc, all margin top centerings break

How about trying this in routePathView.scss
```
    .flexRow {
        align-items: center;
    }

    .flexInnerRow {
        align-items: center;
    }
```

Reply:
Those 5 buttons are now in 2 columns. With this change, they should be in 2 rows instead:

flexRow "varustelutiedot" "solmut exceliin" "aikataulu"
flexRow "solmu" "linkki"